Twitch/Twitch.Base/Services/NewAPI/ChannelPointsService.cs

```diff
@@ -63,11 +63,12 @@ public async Task<bool> DeleteCustomReward(UserModel broadcaster, Guid rewardID)
         /// Gets all rewards associated with the broadcaster.
         /// </summary>
         /// <param name="broadcaster">The broadcaster to get rewards for</param>
+        /// <param name="clientOnly">Determines whether to get only the rewards for the client_id provided</param>
```

Comment:
There isn't a client ID provided as part of the request, so this would be better worded as something like:

"Whether to return only rewards manageable by the current client application"

Reply:
makes sense, i have pushed up the resolved code :)